models/demos/t3000/llama2_70b/tt/llama_common.py

```diff
@@ -415,3 +415,28 @@ def check_kv_cache(pt_cache_all, tt_cache_all, generation_start_pos, generation_
             logger.warning(f"KV Cache Failed! PCC value is lower than {pcc}")
             test_passed = False
     return test_passed
+
+
+def get_flash_decode_chunk_size(s):
```

Comment:
I believe flash decode v2 does not need this function anymore

Reply:
I fixed this in a commit and squashed everything into two commits